python/tank/template.py

```diff
@@ -396,26 +398,24 @@ class TemplatePath(Template):
     """
     Class for templates for paths.
     """
-    def __init__(self, definition, keys, root_path, name=None):
+    def __init__(self, definition, keys, root_path, name=None, root_paths_all_os=None):
```

Comment:
Sorry, now they both feel rather clunky to me! I think it's when they have three underscores, it just feels really long!

`per_platform_roots`?